app/controllers/api/v1/registrations/registrations_controller.rb

```diff
@@ -135,11 +135,12 @@ def validate_payment_ticket_request
   end
 
   def payment_ticket
-    donation = params[:donation_iso].to_i || 0
-    amount_iso = @competition.base_entry_fee_lowest_denomination
-    currency_iso = @competition.currency_code
+    iso_donation_amount = params[:iso_donation].to_i || 0
```

Comment:
What about renaming the `params` key as well?

Reply:
In the `payment_denomination` endpoint below, you are actually using `iso_donation_amount` in `params` as well :upside_down_face:

Reply:
Fixed, and fixed in routes.js.erb